currency.md

```diff
@@ -3,7 +3,7 @@
 ## api link
 
 ```javascript
-let url = `https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest/currencies/${currency}.json`
+let url = `https://cdn.jsdelivr.net/npm/@fawazahmed0/currency-api@2024-04-10/v1/currencies/${currency}.json`
```

Comment:
keep it at latest, as fixed date will show the value to that specific date :

`https://cdn.jsdelivr.net/npm/@fawazahmed0/currency-api@latest/v1/currencies/${currency}.json`